C/CUTEst/build_tarballs.jl

```diff
@@ -1,38 +1,50 @@
 # Note that this script can accept some limited command-line arguments, run
 # `julia build_tarballs.jl --help` to see a usage message.
-using BinaryBuilder
+using BinaryBuilder, Pkg
 
 name = "CUTEst"
-version = v"2.0.4"
+version = v"2.0.5" # <-- This is a lie, we're bumping to 2.0.5 to create a Julia v1.6+ release with experimental platforms
 
 # Collection of sources required to build ThinASLBuilder
 sources = [
-    ArchiveSource("https://github.com/ralna/ARCHDefs/archive/v2.0.4x.tar.gz",
-                  "c9465129952d10a4e6fd049c4741b6e968421c1825c8fc924c1402042e2edb50"),
-    ArchiveSource("https://github.com/ralna/SIFDecode/archive/v2.0.3.tar.gz",
-                  "3a4aa817e1bf4e3595d0e4378da6172b65f02861f3a7c39f9da632a5cc31b1b2"),
-    ArchiveSource("https://github.com/ralna/CUTEst/archive/v2.0.3.tar.gz",
-                  "d21a65c975302296f9856c09034cf46edc5da34b6efd96eed6cc94af6d2c8a55"),
+    GitSource("https://github.com/ralna/ARCHDefs.git" ,"d4aa50f72626130f0e4fb6c8d31c622889a0ebbb"),
+    GitSource("https://github.com/ralna/SIFDecode.git","affd441e93bd41f076239df2f4237fb13278f6a6"),
+    GitSource("https://github.com/ralna/CUTEst.git"   ,"e6aff163eafd9424c70dfaf64a287252221cf597"),
 ]
 
 # Bash recipe for building across all platforms
 script = raw"""
 echo "building for ${target}"
 
 # setup
-mkdir -p ${prefix}/libexec
 mkdir -p ${bindir}
 mkdir -p ${libdir}
-cp -r ARCHDefs-2.0.4x ${prefix}/libexec/
-cp -r SIFDecode-2.0.3 ${prefix}/libexec/
-cp -r CUTEst-2.0.3 ${prefix}/libexec/
-export ARCHDEFS=${prefix}/libexec/ARCHDefs-2.0.4x
-export SIFDECODE=${prefix}/libexec/SIFDecode-2.0.3
-export CUTEST=${prefix}/libexec/CUTEst-2.0.3
+
+cp -r ARCHDefs ${prefix}
+cp -r SIFDecode ${prefix}
+cp -r CUTEst ${prefix}
```

Comment:
These directories end up in the final tarball (including their `.git/` subdirectories, which looks rather weird).  Is this really what you want

Reply:
Yes, it's how @dpo uses this artifact for CUTEst.jl.
https://github.com/JuliaSmoothOptimizers/CUTEst.jl/blob/main/src/CUTEst.jl#L77-L79